.github/workflows/go-ci.yml

```diff
@@ -0,0 +1,12 @@
+on:
+  workflow_call:
+
+jobs:
+  package:
+    # Package only if branch is main or tag
+    if: github.ref == 'refs/heads/main' || contains(github.ref, 'refs/tags/')
+    uses: polyflix/.github/.github/workflows/package-image.yml@main
+    with:
+      artifact-id: build
+      artifact-download: true
+      artifact-download-dir: target
```

Comment:
You didn't upload any artifacts before this step so the `package-image` workflow will fail.

to solutions to solve it:
1. If your Dockerfile builds your project you have to remove the inputs:
```yaml
    with:
      artifact-id: build #remove  
      artifact-download: true #remove
      artifact-download-dir: target #remove
```
2. If your Dockerfile doesn't build the project you have to add an extra job to create a build artifact. You can refer to the build job in the `node-ci` workflow.